js/pop-window.js

```diff
@@ -0,0 +1,65 @@
+  {
+    heading: 'Multi-post Stories 1',
+    image: 'background/Img Placeholder.png',
+    alternative: 'Project',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum has been the industry`s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['html', 'css', 'ruby', 'bootstrap'],
+  },
+  {
+    heading: 'Multi-post Stories 2',
+    image: 'background/Img Placeholder.png',
+    alternative: 'Project',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum has been the industry`s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['html', 'css', 'ruby', 'bootstrap'],
+  },
+  {
+    heading: 'Multi-post Stories 3',
+    image: 'background/Img Placeholder.png',
+    alternative: 'Project',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum has been the industry`s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['html', 'css', 'ruby', 'bootstrap'],
+  },
+  {
+    heading: 'Multi-post Stories 4',
+    image: 'background/Img Placeholder.png',
+    alternative: 'Project',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum has been the industry`s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['html', 'css', 'ruby', 'bootstrap'],
+  },
+];
+
+window.addEventListener('DOMContentLoaded', () => {
+  arrProject.forEach((project) => {
+    const div = document.createElement('div');
+    div.classList.add = 'item';
+    div.innerHTML = `<div class="item">
+                        <div class="project-card">
+                            <img src="${project.image}" alt="${project.alternative}">
+                            <div class="font-change">
+                                 <h3 class="card-head">${project.heading}</h3>
+                                 <p class="description">${project.description}</p>
+                                 <ul class="card-technologies">
+                                    ${project.technologies.map((tech) => `<li>${tech}</li>`).join('')}
+                                 </ul>
+                                <button class="see-project" type="button">See Project</button>              
+                            </div>
+                        </div>
+                    </div>`;
+    document.querySelector('.wrapper').appendChild(div);
+  });
+
+  const popBtn = document.querySelectorAll('.see-project');
+  const techProject = document.querySelector('.pop-container');
+
+  popBtn.forEach((popBtn) => {
+    popBtn.addEventListener('click', () => {
+      techProject.style.display = 'block';
+    });
+  });
+
```

Comment:
- [ ] You might want to consider populating your popup using your array too. So each time you click the See project button, you need to update the information of your popup with the info from the right project.
- An option could be to use an id so it will allow you to get the information of the right project object in your array and use them to fill or update the content of your popup.  How to get that id? Well, this part: 

https://github.com/NataliaPoletaeva/Portfolio/blob/2935342f858c089906c74d710633a6ea7acab2c0/js/pop-window.js#L32-L46

You are using a forEach, that method could give you a index for each step `forEach((element, index) => { ... } )` [Source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) and give that index to the button as a id or data-id(I recommend the second) because the id's are unique. How to get the data-id?

https://github.com/NataliaPoletaeva/Portfolio/blob/2935342f858c089906c74d710633a6ea7acab2c0/js/pop-window.js#L55-L59

Your popBtn is your button. You could use `popBtn.dataset.id` and it returns the id that you need. 
(This is a suggestion but you could implement a different solution.) 😉